src/components/Back/Back.tsx

```diff
@@ -24,10 +24,10 @@ export interface BackProps {
 
 const BackLink = styled.a`
   ${fonts}
-​
+  ​
   --font-size: 16px;
-​
-  font-family: 'Futura Std', sans-serif;
+
+  font-family: 'VamalakaFutura', Arial, Helvetica, sans-serif;
```

Comment:
I think we should just use 'Futura Std'. Or should we really add vamalaka.

Reply:
Thanks, I just use it so that I can see that the fonts are working fine